### PR TITLE
fixed chat room link

### PR DESCRIPTION
**Pull request by Mannuel25**
# Pull Request

## Description

I added the chat room link to the [index file](https://github.com/nhcommunity/homepage/blob/235da2703edab21dd5b267561a6486bbed26c80c/index.html#L40).

## Related Issue

#27 

Closes #XXXXX

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,8 +52,9 @@ <h3>
           community, as well as
           <b><a href="https://www.eddiehub.org/">EddieHub</a></b> by providing
           further support to any who need it. We have a staff of moderators who
-          share this vision, come meet them here (link) and check out some of
-          their amazing projects!
+          share this vision, come meet them <a href="https://chat.nhcarrigan.com" 
+          target="_blank" rel="noopener noreferrer">in our server</a> 
+          and check out some of gittheir amazing projects!
```
Comment (nhcarrigan) [inline index.html:57]:
```suggestion
          and check out some of their amazing projects!
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
One small typo fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Comment by Mannuel25** [comment]
I've made the necessary changes.

**Review by nhcarrigan — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🙂

**Pull request merged**